CHANGES/1026.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Joining URLs with empty segments has been changed
+to match RFC3986 -- by :user:`commonism` and :user:`youtux`
```

Comment:
Sphinx has a dedicated role for RFCs.
```suggestion
to match :rfs:`3986` -- by :user:`commonism` and :user:`youtux`.
```

Reply:
Also, it would be a good idea to include more high-level context for the end users, that is not a reference to elsewhere but is more specific to what the change influences.